topics/02_Controls/igMap/03_Configuring/01_Features/01_igMap_Configuring_Visual_Features.md

```diff
@@ -217,6 +217,12 @@ $("#map").igMap({
 });
 ```
 
+###<a id="map-tooltips-sample">Sample
```

Comment:
@dafo @ig-ealbert I guess that should have been caught by @zdrawku Please be careful when reviewing